core/services/workflows/metering/metering.go

```diff
@@ -194,31 +185,62 @@ func (r *Report) Deduct(ref string, amount int64) error {
 	return nil
 }
 
-// GetAvailableForInvocation returns the amount of credits that can be used based on the available credit balance.
-// This is determined by dividing unearmarked local credit balance by the number of potential concurrent calls.
-func (r *Report) GetAvailableForInvocation(openConcurrentCallSlots int) (int64, error) {
+func (r *Report) CreditToSpendingLimits(
+	info capabilities.CapabilityInfo,
+	amount decimal.Decimal,
+) []capabilities.SpendLimit {
+	// TODO: https://smartcontract-it.atlassian.net/browse/CRE-463 handle multiple spend types
+	if len(info.SpendTypes) > 0 {
+		spendType := info.SpendTypes[0]
+
+		// use rate card to convert capSpendLimit to native units
+		spendLimit := r.balance.ConvertFromBalance(string(spendType), amount)
+
+		// check to see if metering mode is active
+		if r.balance.meteringMode {
+			return nil
+		}
+
+		return []capabilities.SpendLimit{{SpendType: spendType, Limit: spendLimit.StringFixed(defaultDecimalPrecision)}} // TODO: should we apply rounding? maybe take only the int part?
```

Comment:
```suggestion
		return []capabilities.SpendLimit{{SpendType: spendType, Limit: spendLimit.StringFixed(defaultDecimalPrecision)}}
```

Remove TODO comment, as rounding is already added